steam_app/controllers/GamesViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ViewController.swift
+//  steam_app
+//
+//  Created by Khumoyiddin on 28/12/22.
+//
+
+import UIKit
+
+class GamesViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let nib = UINib(nibName: "GamesTableViewCell", bundle: nil)
+        
+        tableView.register(nib, forCellReuseIdentifier: "GamesTableViewCell")
```

Comment:
It is better not to write id of the tableView as string. It would be much better if you created an extension to the tableviewcell to hold its name and reuse it. This would help you to avoid string mistakes in the future